docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -464,12 +409,12 @@ It should be possible to skip policy validation. This can be seen as similar to
 
 ```sql
 SET FOREIGN_KEY_CHECKS=0;
-SET PLACEMENT_CHECKS=0;
+SET tidb_placement_mode='IGNORE';
 
 CREATE TABLE t3 (a int) PLACEMENT POLICY `mycompanypolicy`;
 ```
 
-If a table is imported when `PLACEMENT_CHECKS` is `OFF`, and the placement policy does not validate, then the same rules of fallback apply as in the case `DROP PLACEMENT POLICY` (where policy is still in use).
+If a table is imported when `tidb_placement_mode='IGNORE'`, and the placement policy does not validate, then the same rules of fallback apply as in the case `DROP PLACEMENT POLICY` (where policy is still in use).
```

Comment:
Do we also mention `tidb_placement_mode='strict'`, since it is not off/on, but an enum.

Reply:
Sure, I'll add.